pyOCD/tools/pyocd.py

```diff
@@ -648,7 +645,7 @@ def handle_clock(self, args):
         else:
             nice_freq = "%d Hz" % freq_Hz
 
-        print "Changed %s frequency to %s" % (swd_jtag, nice_freq)
+        print "Changed frequency to %s" % nice_freq
```

Comment:
I agree @c1728p9, this is the same point I was making about the absolute imports